lms/envs/common.py

```diff
@@ -1495,6 +1495,18 @@
         ],
         'output_filename': 'css/certificates-style-rtl.css'
     },
+    'style-leaner-dashboard': {
```

Comment:
Just want to double check. You really mean "leaner", not "learner"? I don't know if I like the comparative grammar in the word "leaner". What if we got an even "leaner" dashboard in the future?